packages/app-content-pages/src/screens/Publications/Publications.js

```diff
@@ -55,7 +55,7 @@ function Publications({ className = '', data = [], filters = [] }) {
   )
 
   const sidebar = (
-    <Nav ariaLabel={t('Publications.sideBarLabel')}>
+    <Nav aria-label={t('Publications.sideBarLabel')}>
```

Comment:
Not a blocking comment, but is there a test to find this labelled navigation menu?

Reply:
Good point~ I don't think there is yet, so I'll add before merging!